### PR TITLE
Feature/mark completed congratulate

### PR DESCRIPTION
**Pull request by kncaputo**
## What is the change?
- Add functionality for timer completion and "log activity" button, plus initial css and html for saved activity cards.

## What does it fix?
- Refactored startTimer method in class to remove DOM tasks. Combined formatTime and formatRemainingTime function. 

## Is this a fix or a feature?
- Both

## Where should the reviewer start?
- Start by reviewing the methods in Activity.js. Then review the bottom of the main.js file. Also review the bottom of the stylesheet to see the styles for the cards.

## How should this be tested?
- Test by opening index.html and complete an activity and click log activity button.

### CODE REVIEW

**Review by peytonjo — APPROVED** [review]
It's looking so good I cant wait to see the finished product!

**Review by DennClark — APPROVED** [review]
Looks really good!

File: main.js [hunk 1]
```diff
@@ -85,76 +88,83 @@ function triggerAlert(alertNeeded) {
...
-  if (currentActivity.minutes < 10 && currentActivity.minutes > 0) {
-    minutes = `0${currentActivity.minutes}`;
-  }
-  if (currentActivity.minutes === 0) {
-    minutes = '00'
-  }
+  if (seconds < 10 && seconds > 0) {seconds = `0${seconds}`;}
+  if (seconds < 1) {seconds = '00';}
+  if (minutes < 10 && minutes > 0) {minutes = `0${minutes}`;}
+  if (minutes === 0) {minutes = '00'}
```
Comment (lucyconklin) [inline main.js:130]: Multiple if statements in a row is usually a good place to start refactoring. Maybe something like [padStart()]( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) would be a bit cleaner? (caveat: I have not actually used this, I just found it now)

File: index.html [hunk 2]
```diff
@@ -69,7 +69,7 @@ <h2 class="section-title">New Activity</h2>
             <p class="alert-msg">Limit duration to 59 minutes and 59 seconds.</p>
           </div>
           <article class="start-container">
-            <button class="start-activity-btn">START ACTIVITY</button>
+            <button class="start-activity-btn btn">START ACTIVITY</button>
```
Comment (lucyconklin) [inline index.html:72]: It looks like you may have some styles that are shared between all of your buttons. I'd encourage you to pull those out into a the shared `btn` class and use it like you are here.

**Review by lucyconklin — COMMENTED** [review]
Looking Good! I had a few minor suggestions for refactoring. Also, if you are finding some of this time/seconds/minutes stuff tedious, know that you'll mostly be using libraries to help you with time and dates in the future, but for now it is really good to know how it works.

Fun fact, [JS starts counting months at 0](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth#:~:text=The%20getMonth()%20method%20returns,first%20month%20of%20the%20year), so January is month 0. 🤷‍♀️

**Pull request merged**